### ISSUE TITLE
Support for ping/pong frames

### ISSUE THREAD

**Issue by crowlKats** [ricea/websocketstream-explainer#10]
The WebSocket API is quite awkward in regard to ping/pong frames. It would be great if WSS could support them. Has there been any thoughts regarding that?

**Comment by ricea** [comment]
It's tricky to know what to do with ping/pong frames, which is probably why there hasn't been success in exposing them to JavaScript. They can arrive in the middle of a message, so they can be considered "out of band".

This is analogous to how TCP keepalive messages are not directly exposed to user space APIs. They are only useful in determining whether the lower layers of the stack are alive; if you need to know whether the application layers are alive you still need application-level acks.

In conclusion, I don't know a good way to support them, and it's not an immediate priority.
